.github/workflows/crawler.yml

```diff
@@ -1,34 +1,12 @@
+# Auto-generated by Cimas: Do not edit it manually!
+# See https://github.com/metanorma/cimas
 name: Crawler
 
 on:
   push:
   schedule:
-    - cron: '0 15 * * *'
-  workflow_dispatch:
```

Comment:
Why has the workflow_dispatch been removed?

Reply:
added back